pootle/apps/pootle_store/views.py

```diff
@@ -138,10 +138,7 @@ def get_step_query(request, units_queryset):
                     checks = request.GET['checks'].split(',')
                 elif 'category' in request.GET:
                     category_name = request.GET['category']
-                    try:
-                        category = get_category_id(category_name)
-                    except KeyError:
-                        raise Http404
+                    category = get_category_id(category_name)
```

Comment:
@unho this exception can't be raised because `get_category_id()` returns `None` in case there is no `category_name` in `CATEGORY_IDS` dictionary. I asked if we validate it anywhere.